front/components/ContentNodeTree.tsx

```diff
@@ -178,8 +177,9 @@ function ContentNodeTreeChildren({
               selectedNodes
                 ? {
                     disabled: parentIsSelected || !setSelectedNodes,
-                    checked: checkedState,
-                    onChange: (checked) => {
+                    isPartial: checkedState === "partial",
+                    checked: checkedState === "checked",
```

Comment:
How does those two works? What is `isPartial` is true and `checked` is true? Don't we want one unique props to control the entire state?

Reply:
If both are true then `isPartial` "takes over". But yeah agree, might want one single prop, will change that

Reply:
👍